### PR TITLE
{lib}[GCCcore/11.2.0] GLib v2.69.1, NSPR v4.32, NSS v3.69

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/e0756c379e351e843b961bac8771b486 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 41 out of 42 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/404791f3acdb2c5996e93d7ec34c6df9 for a full test report.

**Comment by Micket** [comment]
So.. it looks like it also needs PCRE (1) ?

**Comment by Micket** [comment]
Yup I can see from my buildlog that it did download and build PCRE as a subproject. Ouchie

**Comment by Micket** [comment]
@boegel I think you introduced PCRE2 as a dep last time GLib was updated. It doesn't seem to use PCRE2 at all. 2.68.2 seems to havebundled PCRE 1 inside it's sources, and that's why it sort of worked.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/7c6e272ceb15adf6a60b9f29e294c9c9 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/04c2bc8037ae7e4c8635f6ce1842245d for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/0c2a8971c7eada0d2ea2e86a5f00fa67 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 36 out of 42 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/922c4b05ac1c73bc42bb9d73cad84888 for a full test report.

**Comment by boegel** [comment]
@verdurin Rust failed because it can't find OpenSSL because of https://github.com/easybuilders/easybuild-easyblocks/issues/2525?

**Comment by verdurin** [comment]
@boegel yes, I think so.

**Comment by Micket** [comment]
I'm not sure what to make of the NSS build error there. Overzealous warning or actual problem? Perhaps incompatibility with a very new glibc or something?

File: easybuild/easyconfigs/n/NSS/NSS-3.65-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'NSS'
+version = '3.65'
```
Comment (branfosj) [inline easybuild/easyconfigs/n/NSS/NSS-3.65-GCCcore-11.2.0.eb:4]: There is a 3.69 available (https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_69_RTM/src/). Is there a reason for sticking to 3.65?

Comment (Micket) [reply easybuild/easyconfigs/n/NSS/NSS-3.65-GCCcore-11.2.0.eb]: Homepage still listed 3.65 as the stable release but i think the homepage was just oudated.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/b10faae7da0f007ec1dbb85e7b8dfd18 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in total)
bear-pg0211u14b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/e4cba6d98ec19b931ff19c8d93b982bd for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/0fe3bc9161466d3fd0a812b162ecbc3e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u14b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/dc0db074199fa0ff811d9f3e809148f1 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/53a8857548eb81673a23294f2b01ad1d for a full test report.

**Comment by Micket** [comment]
@verduring Can you please try again with this NSS patch?

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/0e33115117e19f147ffbf5e011634478 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/4b6611aaf88034c5df84ef3c2ecc5156 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/206f669b974102b612367f3dafce61dc for a full test report.

**Comment by SebastianAchilles** [comment]
> @verduring Can you please try again with this NSS patch?

@verdurin Can you test again this PR on your system?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6a61f180e3579632d422e860a857832b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 7 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/2312bbf9b58bb1ec0f5ce0986b6910f7 for a full test report.

**Comment by Micket** [comment]
Lockfile conflict?

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @Micket!

**Pull request merged**